src/components/BulkMemberManager/BulkMemberManager.js

```diff
@@ -119,7 +123,7 @@ const BulkMemberManager = ({
                                     ? removeEntity(pendingChanges, entity)
                                     : addEntity(pendingChanges, entity)
                             )
-                            toggleSelected(entity.id)
```

Comment:
This was causing a subtle bug because it would either add or remove the selected result from the selected results set, depending on the current state of the checkbox. It now feels very obvious to point out that this was wrong, but it took a while to get to that conclusion 😅